manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```diff
@@ -212,7 +212,10 @@ spec:
         securityContext:
```

Comment:
Consider adding seccompProfile to ensure compliance with the Restricted Pod Security Standards (https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted).
```
          seccompProfile:
            type: RuntimeDefault
```

Reply:
We haven't used this elsewhere in Istio (and if we do it here, we should be consistent with the rest of Istio) due to the fact `RuntimeDefault` is basically "literally anything the platform decides", which could be somethings more restrictive than what we allow. There are runtimes where the default doesn't even allow pthread_create (??) for instance

Reply:
If we go down that path, yeah - we might as well just template `seccompProfile`, default it to our current, and allow people to roll the dice with their own seccompProfiles.